com.github.vikdevelop.photopea_app.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea
+    - install -Dm755 -t /app/share/applications $FLATPAK_ID.desktop
```

Comment:
I was saying to remove the .desktop in this repository and use the one from upstream (like it was before) but use `rename-desktop-file` to rename it.

btw this won't work anyway since at that point the file `com.github.vikdevelop.photopea_app.desktop` can't be found

(see https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html )